### ISSUE TITLE
Cannot edit page

### ISSUE THREAD

**Issue by devism** [bleutzinn/grav-plugin-editable#12]
I'm using content tools and when I go into edit mode the only editable area is at the bottom of the page and it seems to be a 'p' element that is not part of the original page. Inspecting the page I can see that a div with  `data-editable data-name="editable___menu"` is wrapping my content but I can't edit any text on the html elements. 

my editable.yaml :

enabled: true
active_admin: false
editor: contenttools
editable_self: true

**Comment by bleutzinn** [comment]
Editable should not be used with modular pages and assuming from your page name ("menu") you are trying to edit a modular page. Please see [#8](https://github.com/bleutzinn/grav-plugin-editable/issues/8).

In future versions of editable the use on modular pages will not be possible at all to avoid the confusion.

As a best practice I recommend to disable site wide use of `editable_self` but enable `editable_self` on some normal (!) pages. When using ContentTools, as you do, make sure the page content is HTML only. ContentTools will simply skip all non HTML content.

**Comment by devism** [comment]
Actually the menu page is not a modular page but I got it working. What I had to do to make it work  was simplifying most of the html. It looks like content tools is not compatible with certain elements. I had several div's, sections, etc and I had to simplify to to just`ul` an `li`. Some of the html elements had onclick functions so maybe that was part of the problem? It's kind of confusing since I couldn't find documentation on what html elements are compatible for editing.
